docs/platform.md

```diff
@@ -112,6 +112,20 @@ Returns a boolean which defines if device is a TV.
 
 ---
 
+### `isVision`
+
+```tsx
+static isVision: boolean;
+```
+
+Returns a boolean which defines if device is an Apple Vision.
```

Comment:
I would also add that this is true only in non-compatibility mode. If we are running on Apple Vision Pro (Designed for iPad) visionOS will return false and isPad will be true